src/util/ped.hpp

```diff
@@ -23,6 +23,10 @@ namespace big::ped
 		return true;
 	}
 
+	bool is_player_in_cutscene(Ped player) { return NETWORK::NETWORK_IS_PLAYER_IN_MP_CUTSCENE(player); }
+
+	bool is_player_in_interior(Ped player) { return (INTERIOR::GET_INTERIOR_FROM_ENTITY(player) > 0) ? true : false; }
```

Comment:
Nit: we can get rid of the ternary and just do `return (INTERIOR::GET_INTERIOR_FROM_ENTITY(player) != 0)`